src/models/converters/transactions/tests/transfer_type_checks.rs

```diff
@@ -0,0 +1,213 @@
+        to,
+        value: Some(String::from("0")),
+        data: Some(String::from("0xa9059cbb000000000000000000000000938bae50a210b80ea233112800cd5bc2e76443000000000000000000000000000000000000000000000000000000000000000466")),
+        data_decoded: Some(DataDecoded {
+            method: String::from("transfer"),
+            parameters: Some(vec!(
+                Parameter {name: String::from("to"), param_type: String::from("address"), value: String::from("0x938bae50a210b80EA233112800Cd5Bc2e7644300")},
+                Parameter {name: String::from("value"), param_type: String::from("uint256"), value: String::from("1126")},
+            ))
+        }),
+        operation: Some(Operation::CALL),
+        gas_token: Some(String::from("0x0000000000000000000000000000000000000000")),
+        safe_tx_gas: Some(47810),
+        base_gas: Some(0),
+        gas_price: Some(String::from("0")),
+        refund_receiver: Some(String::from("0x0000000000000000000000000000000000000000")),
+        nonce: 175,
+        execution_date: Some(expected_date),
+        submission_date: expected_date,
+        modified: Some(expected_date),
+        block_number: Some(6954173),
+        transaction_hash: Some(String::from("0xc76ee22b0ab2785c5c8e93d029acf5643a8fccabcc3f223704bdbfc2af8193ae")),
+        safe_tx_hash: String::from("0xcddc60c644e85ee3bff84204380a86578d3b62f8dd05a86de40e7e662012caf5"),
+        executor: Some(String::from("0xF2CeA96575d6b10f51d9aF3b10e3e4E5738aa6bd")),
+        is_executed: true,
+        is_successful: Some(true),
+        eth_gas_price: Some(String::from("1000000000")),
+        gas_used: Some(80507),
+        fee: Some(String::from("80507000000000")),
+        origin: None,
+        confirmations_required: Some(3),
+        confirmations: Some(vec!(
+            Confirmation {
+                owner: String::from("0x65F8236309e5A99Ff0d129d04E486EBCE20DC7B0"),
+                submission_date: Some(expected_date),
+                transaction_hash: None, 
+                signature_type: String::from("EOA"),
+                signature: Some(String::from("0x6722a6772c15dd72851593c69e521729be5f6e86cbe6881498885b6efb02f6a61638976cf12bb113d8a1f7cea776e9d5c4949dde7d8015ca03e433acc6f9435b1b")),
+            },
+            Confirmation {
+                owner: String::from("0xBEA2F9227230976d2813a2f8b922c22bE1DE1B23"),
+                submission_date: Some(expected_date),
+                transaction_hash: None, 
+                signature_type: String::from("EOA"),
+                signature: Some(String::from("0x75aa624315ed8a72e1fcf396fe8d7a50d6d71f18c3b6bddb025bf426b7e4de3f52ead25cef15c9b35feaeefb2b81f2b5333ed949035f948fbe0a9aec56f7ac5b1b")),
+            },
+            Confirmation {
+                owner: String::from("0xF2CeA96575d6b10f51d9aF3b10e3e4E5738aa6bd"),
+                submission_date: Some(expected_date),
+                transaction_hash: None, 
+                signature_type: String::from("EOA"),
```

Comment:
I was wondering if we should pass this on to the clients. I think Uxio told us we could ignore it, but not sure if sometime down the road this will become relevant.

Reply:
the confirmation_type can be ignored, not the signature_type

Reply:
Also it is currently not passed along, right? This is something that needs to be evaluated after 1.0.0 (as it might be required for the web app)